lighthouse-core/test/gather/gatherers/offline.js

```diff
@@ -19,57 +19,22 @@
 
 const OfflineGather = require('../../../gather/gatherers/offline');
 const assert = require('assert');
+const tracingData = require('../../fixtures/traces/network-records.json');
 let offlineGather;
 
-describe('HTTP Redirect gatherer', () => {
+describe('Offline gatherer', () => {
   // Reset the Gatherer before each test.
   beforeEach(() => {
     offlineGather = new OfflineGather();
   });
 
-  it('returns an artifact', () => {
-    return offlineGather.afterPass({
-      driver: {
-        sendCommand() {
-          return Promise.resolve();
-        },
-        evaluateAsync() {
-          return Promise.resolve({offlineResponseCode: 200});
-        }
-      }
-    }).then(_ => {
-      assert.deepEqual(offlineGather.artifact, {offlineResponseCode: 200});
-    });
+  it('returns an artifact set to -1 when offline loading fails', () => {
+    offlineGather.afterPass({url: 'https://do-not-match.com'}, tracingData);
```

Comment:
actually, if the gatherer doesn't do any navigation these would be good tests, just processing static network records :)